pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -242,12 +253,7 @@ func linkNodeJsPackage(ws pkgWorkspace.Context, root string, pkg *schema.Package
 
 // printNodeJsImportInstructions prints instructions for importing the NodeJS SDK to the specified writer.
 func printNodeJsImportInstructions(w io.Writer, pkg *schema.Package, options map[string]interface{}) error {
-	var importName string
-	if info, ok := pkg.Language["nodejs"].(nodejs.NodePackageInfo); ok && info.PackageName != "" {
-		importName = info.PackageName
-	} else {
-		importName = cgstrings.Camel(pkg.Name)
-	}
+	importName := cgstrings.Camel(pkg.Name)
```

Comment:
Why has this changed? Don't we still want the import name from `nodejs`?

Reply:
This was buggy and incorrect.  `PackageName` is documented in `pulumi.json` as "NPM package name (includes @namespace)".  Same with the test you're commenting on below.  Maybe I should pull this out into a separate PR :thinking:

Reply:
Pulled it out into https://github.com/pulumi/pulumi/pull/18933.